internal/agent/service/service_windows.go

```diff
@@ -91,12 +64,10 @@ func (d *daemon) run() {
 
 		switch exitCode {
 		case api.ExitCodeRestart:
-			log.Info("agent process exited with restart exit code. restarting agent process...")
+			log.Info("child process requested restart")
```

Comment:
calling the service child process (previously known as "agent process") "child" to distinguish btw agent service process and child one